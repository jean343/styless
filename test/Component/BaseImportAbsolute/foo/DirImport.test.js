import React from 'react';
import renderer from 'react-test-renderer';
import styled from 'styled-components';
import 'jest-styled-components';

test('It should find the base import in sub folders', () => {
    const Div = styled.div`
		color: @base-color;
		.base-mixin;
	`;
    expect(renderer.create(<Div/>).toJSON()).toMatchSnapshot();
});