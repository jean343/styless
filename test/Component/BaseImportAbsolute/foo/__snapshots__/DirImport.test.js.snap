// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should find the base import in sub folders 1`] = `
.c0 {
  color: #b5483d;
  background: #39688f;
}

<div
  className="c0"
/>
`;
